guides/common/modules/proc_configuring-project-for-ansible-compliance-policy-deployment.adoc

```diff
@@ -0,0 +1,15 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-{project-context}-for-ansible-compliance-policy-deployment"]
+= Configuring {Project} for Ansible compliance policy deployment
+
+If you want to use Ansible to deploy compliance policies, configure {Project} for Ansible compliance policy deployment.
+
+.Procedure
+. Import the `theforeman.foreman_scap_client` Ansible role.
```

Comment:
This lacks an entry point: It's either scp'ing the Ansible role onto Foreman Server, or, if you have installed `ansiblerole-foreman_scap_client` via RPM, then it's Web UI, Ansible > Roles.

Reply:
https://github.com/theforeman/foreman-documentation/issues/4214